.github/workflows/validate-solutions.yml

```
@@ -0,0 +1,168 @@
+  validate:
+    runs-on: [self-hosted, Windows, X64]
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Set up Rust
+      uses: actions-rust-lang/setup-rust-toolchain@v1
+      with:
+        toolchain: stable
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)